web/client/actions/map.js

```diff
@@ -143,9 +152,10 @@ function updateVersion(version) {
     };
 }
 
-function initMap() {
+function initMap(location) {
     return {
-        type: INIT_MAP
+        type: INIT_MAP,
+        location
```

Comment:
Why this addition?

Reply:
good catch, it was there from a first implementation and it is no longer used.
going to fix it